app/forms/work_form.rb

```diff
@@ -6,6 +6,7 @@
 class WorkForm < DraftWorkForm
   validates :abstract, :access, :title, presence: true, allow_nil: false
   validates :keywords, length: { minimum: 1, message: 'Please add at least one keyword.' }
+  validate :keywords_unique
   validates :attached_files, length: { minimum: 1, message: 'Please add at least one file.' }
```

Comment:
Before doing this custom validator method, I played around with the `unique: true` declaration on the `validates :keywords` method (which is a Reform validator: https://github.com/trailblazer/reform-rails/blob/master/lib/reform/form/validation/unique_validator.rb) but couldn't get it to work as expected (either on it's own or with a scope).